lib/reactotron-react-native-mmkv/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+  "extends": "../../tsconfig.base.json",
+  "compilerOptions": {
+    "allowJs": false,
+    "declaration": true,
+    "declarationDir": "dist/types",
+    "rootDir": "src",
+    "emitDeclarationOnly": true,
+    "emitDecoratorMetadata": true,
+    "allowSyntheticDefaultImports": true,
+    "experimentalDecorators": true,
+    "module": "es2015",
```

Comment:
Any reason not to just go `esNext`?

Reply:
This was copy and pasted over from `reactotron-mmkv`.

There is a lot of legacy `tsconfig.json` settings and rollup stuff that could be simplified, but I think for now it's worth keeping the same until we have a cohesive strategy across all the modules

Reply:
Eventually, it would be nice to have all of these extend from `@tsconfig/react-native/tsconfig.json` or something like that and drop rollup.